pysat/__init__.py

```diff
@@ -85,10 +85,12 @@
         data_dir = '/home/travis/build/pysatData'
         params['data_dirs'] = [data_dir]
 
-    print(''.join(("\nHi there!  Pysat will nominally store data in the "
-                   "'pysatData' directory at the user's home directory level. "
-                   "Set `pysat.params['data_dirs']` equal to a path that "
-                   "specifies a top-level directory to store science data.")))
+    print(''.join(("\nHi there!  pysat will nominally store data in a ",
+                   "'pysatData' directory which needs to be assigned. ",
+                   "Please run `pysat.params['data_dirs'] = path` where path ",
```

Comment:
Aren't you supposed to do: `pysat.utils.set_data_dir(path)`?

Reply:
That is gone now that pysat moved to pysat.params.